models/reviews.model.js

```diff
@@ -66,19 +66,28 @@ return rows.map((review) => {
           return review;
         })
     });
-// const allReviews = db.query(`SELECT * FROM reviews`)
-// const allComments = db.query(`SELECT * FROM comments`)
-// return Promise.all([allReviews, allComments]).then(([reviewsData, commentsData])=>{
-//     const everyReview=reviewsData.rows;
-//     const everyComment=commentsData.rows;
-//     everyReview.forEach((review)=>{
-//       review.comment_count=0;
-//       everyComment.forEach((comment)=>{
-//         if(comment.review_id===review.review_id){
-//           review.comment_count++;
-//         }
-//       })
-//     })
-//   return everyReview;
-// })
 };
+
+exports.fetchReviewCommentsById=(id)=>{
+//  return db.query(`SELECT * FROM comments 
```

Comment:
Remove unncessary comments. Comments should only be left in if they are explaining what the code is doing to make it clearer. Past code is not needed and only confuses people